cas-server-webapp/src/main/webapp/WEB-INF/webflow/login/login-webflow.xml

```diff
@@ -196,6 +196,7 @@
         <transition on="AccountNotFoundException" to="generateLoginTicket"/>
         <transition on="AccountDisabledException" to="casAccountDisabledView"/>
         <!-- Customized exceptions for OSF -->
+        <transition on="InvalidVerificationKeyException" to="casInvalidVerificationKeyView" />
```

Comment:
Unlike `FailedLoginException` that is redirected to `generateLoginTicket` (and then causing the infinite loop), `InvalidVerificationKeyException` simply lands the user on a dedicated exception page.